src/PrestaShopBundle/Resources/config/services/adapter/country.yml

```
@@ -23,3 +23,9 @@ services:
 
   prestashop.adapter.country.repository.country_repository:
     class: 'PrestaShop\PrestaShop\Adapter\Country\Repository\CountryRepository'
+
+  prestashop.adapter.country.command_handler.create_country_handler:
```

```suggestion
  prestashop.adapter.country.command_handler.add_country_handler:
```

We usually define the service names based on the class names